tests/config/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+Testing configuration for superset-patchup
+"""
+
+# DRAGONS: Superset likes to configure initialization via the config -
+# so we only import (and init) the -patchup module here so we're sure
+# superset is already initializing.
+# This also mimics our deployments where we add_ketchup inside the
+# custom config itself.
+import superset_patchup
```

Comment:
Basically the extra module is needed to allow lazy-loading of the -patchup libraries, want to make sure that superset is always around before -patchup gets initted.